functions/go-mongo/README.md

```diff
@@ -0,0 +1,59 @@
+# Mongo + Go example
+
+Example to demonstrate the possibility to connect Serverless Functions to MongoDB.
+
+For this example, [Scaleway Console](https://console.scaleway.com/) will be used for deployment.
+
+> [!WARNING]
+> This is a basic sample that does not use certificate for authentication, not recommended for production.
+
+## Requirements
+
+- Mongo DB created [documentation](https://www.scaleway.com/en/docs/managed-mongodb-databases/quickstart/#how-to-create-a-database-instance)
+
+## Step 1 - Mongo
+
+After Mongo DB creation, in the console find the public endpoint of the database, it should look lile: `<scw_database_id>.mgdb.<scw_region>.scw.cloud`
+
```

Comment:
Also, it lacks the info that the MongoDB must be publicly accessible. Inside a private network, Mongo DB won't be reachable from the container as of today.

Reply:
fixed